pkg/p2p/kadcast/router.go

```diff
@@ -235,32 +250,81 @@ func (router *Router) sendNodes(receiver Peer) {
 // BroadcastPacket sends a `CHUNKS` message across the network
 // following the Kadcast broadcasting rules with the specified height.
 func (router *Router) broadcastPacket(height byte, tipus byte, payload []byte) {
```

Comment:
I tried to avoid make huge functions as much as I could since will be easier to mantain and read. 
Anyway, the content is correct. Just leaving this for a possible refactor in the future.

Reply:
The main effort with this and previous PRs was to get compatible with the protocol spec. A mode complete refactoring will be done later on.